torch/onnx/symbolic_helper.py

```diff
@@ -867,6 +881,40 @@ def _set_training_mode(training_mode):
     global _training_mode
     _training_mode = training_mode
 
+# Symbolic function state.
+class SymbolicFunctionState(object):
```

Comment:
I'm sorry to contradict the other reviewer, but please don't write `(object)`. See earlier comment for rationale.

Reply:
adjusted